.eslintrc

```diff
@@ -1,5 +1,5 @@
 {
-  "parser": "babel-eslint",
+  "parser": "@babel/eslint-parser",
```

Comment:
@sherakama eslint wasn't working in my PhpStorm. I found that babel-eslint is deprecated so I updated to this new one and now it's working for me.